docs/src/css/codes.css

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+.codes .name {
+  cursor: alias;
+}
+
+.codes .name:hover {
+  background: var(--ifm-font-color-base-inverse);
```

Comment:
Not a fan of making the background seemingly disappear when hovering. Either it should go slightly darker, or just not change at all IMO

Reply:
Yeah, me too.  It was a quick-win solution to cope with both light and dark themes.  I'll look into it further when I get a chance.

Reply:
Went for a more subtle approach.  Any thoughts?